src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1454,8 +1454,8 @@ services:
     tags:
       - { name: form.type }
 
-  form.type.sell.product.specification.specifications_type:
-    class: 'PrestaShopBundle\Form\Admin\Sell\Product\Specification\SpecificationsType'
+  form.type.sell.product.details.details_type:
```

Comment:
If you want to change those service definitions, can you also change also the service Id and mark them as private

Reply:
it could be done in dedicated PR, as it would also probably make sense to change them for all the other forms at least in product page. This will grow and cause conflicts, so could we do it next time (as this is already approved)?

Reply:
This PR will never happen sadly, I think we must do it now. This is progressive work so why not doing it now instead of waiting ?

Reply:
just because the PR has a clear scope and its easier to review and test it without unrelated changes. 
But ok I can do it, but Im only changing this one type, not all product types.

Reply:
I meant just for the one you've edited, of course editing all others is completlty out of the scope